lib/http.rb

```diff
@@ -15,7 +15,17 @@
 require "http/response/body"
 require "http/response/parser"
 
-# HTTP should be easy
+# HTTP should be easy.
+#
+# Basic Usage
+#
+# HTTP.post("https://server.com", options)
+#
+# For more detailed documentation and usage notes:
+# @see https://github.com/httprb/http/wiki the http wiki
+#
+# Methods in HTTP class object generally are implemented in the Chainable module.
```

Comment:
I believe it's redundant by different reasons:

1. it's trying to expose implementation details
2. YARD will tell that this module is extended with Chainable module and thus has ... methods
3. It's pretty obvious from the code that one should look into Chainable for methods

Reply:
It was not obvious.  From the example in the README I went to the HTTP class documentation and mostly found a blank page, which was very confusing.  The example use the `HTTP.get` method, but the yard doc had no `get` method documented.

Reply:
Are we talking about the same API doc page?

https://www.rubydoc.info/gems/http/HTTP

---

![image](https://user-images.githubusercontent.com/94782/72532138-2583a380-3873-11ea-96b0-43c3fd747f79.png)

---

Reply:
Yes that looks blank to me when I'm searching for the documentation on `HTTP.get`.  It took me a while to figure out that I needed to look at the `Chainable` module.

Reply:
Ok. But that's not the reason to bloat api docs with info that is already there.

Reply:
The README pretty clearly calls out both [the Wiki](https://github.com/httprb/http/wiki) and the `Chainable` API documentation:

<img width="574" alt="Screen Shot 2020-01-16 at 10 24 02 AM" src="https://user-images.githubusercontent.com/797/72551780-67e4ba80-384a-11ea-8c58-c0ed1cbc0c55.png">

Perhaps we can move the `Documentation` section higher up to the top or something. I'm curious why you might have glossed over both of those links?